gplog/gplog.go

```diff
@@ -108,8 +114,14 @@ func InitializeLogging(program string, logdir string) {
 	}
 
 	createLogDirectory(logdir)
-	timestamp := operating.System.Now().Format("20060102")
-	logfile := fmt.Sprintf("%s/%s_%s.log", logdir, program, timestamp)
+
+	var logfile string
```

Comment:
For consistency with how the logfile prefix is handled, it might make more sense to pull this out into a `GenerateLogFileName` function (putting the existing logic in a `defaultLogNameFunc` function), so that it can be called both here and in any log rotation functions added later.

Reply:
done, pls review